docker-compose.yml

```diff
@@ -5,7 +5,7 @@
 services:
   cvat_db:
     container_name: cvat_db
-    image: postgres:15-alpine
+    image: postgres:15-alpine3.17
```

Comment:
@kirill-sizov  
could you add a comment why this change is needed?

Reply:
Added